src/sources/datadog_agent/mod.rs

```diff
@@ -113,6 +113,9 @@ pub struct DatadogAgentConfig {
 
     /// If this is set to `true`, when log events contain the field `ddtags`, the string value that
     /// contains a list of key:value pairs set by the Agent is parsed and expanded into an object.
+    ///
+    /// Note: This setting introduced in 0.37.0 is incorrectly parsing into an object. This will be
```

Comment:
I imagine you are running this, but we'll just need to regenerate the docs.

Reply:
yep :) compiling 🐌